src/site/content/en/blog/prefers-color-scheme/index.md

```diff
@@ -0,0 +1,725 @@
+---
+title: Hello Darkness, My Old Friend
+subhead: Overhyped or necessity? Learn everything about dark mode and how to support it to benefit your users!
+authors:
+  - thomassteiner
+date: 2019-06-30
+hero: dark.png
+alt: |
+  Web page rendered in the dark mode experience with the Chrome Developer Tools open on the Network tab
+  that shows the CSS file responsible for the (currently not used) light mode experience
+  being downloaded with the lowest priority.
+description: |
+  Many devices now support an operating system wide dark mode or dark theme experience.
+  This post explains how dark mode can be supported on web pages, lists best practices,
+  and introduces a custom element named dark-mode-toggle that allows web developers
+  to offer users a way to override their operating system level preference on specific web pages.
+tags:
+  - post
+  - dark-mode
+  - dark-theme
+  - prefers-color-scheme
+  - color-scheme
+---
+
+## Dark mode before *Dark Mode*
+
+<figure class="w-figure w-figure--inline-right">
+  <img style="height:175px; width:auto;" src="green-screen.jpg" alt="Green screen computer monitor" intrinsicsize="640x480">
+  <figcaption class="w-figcaption">Fig. — Green screen (<a href="https://commons.wikimedia.org/wiki/File:Compaq_Portable_and_Wordperfect.JPG">Source</a>)</figcaption>
+</figure>
+
+We have gone full circle with dark mode.
```

Comment:
May want to consider compressing the intro, this article is easily double the length of most of the others.

Reply:
Fully agree it's long, but I think it's necessary because else people don't understand why it's such a big deal. But as a compromise, I have added a "skip (Flash) intro" link right at the beginning in https://github.com/GoogleChrome/web.dev/pull/1021/commits/078e53c8b0d5eb16e14480d9167052ba2bcbb5ac.

Reply:
I might propose inverting this a bit. Putting the content about the media query at the top, and making the second half of the post about the backstory. Another option would be to split it into two posts. One on the media query and another on the history.

We usually recommend that articles be between 1000 to 1500 words but this one is ~3400. We would like to stick to this policy because there are only a small group of folks doing content reviews and it's hard for them to scale when content is really lengthy. What has tended to happen in the past is really long articles get stuck in the content review stages for a long time and we want to avoid that.

Reply:
> I might propose inverting this a bit. Putting the content about the media query at the top, and making the second half of the post about the backstory. Another option would be to split it into two posts. One on the media query and another on the history.

I don't think this would flow right if inverted. Note that there is a [skip intro](https://github.com/GoogleChrome/web.dev/blob/108545d23a99086dbbad00da794533e4c38c6e50/src/site/content/en/blog/prefers-color-scheme/index.md#dark-mode-before-dark-mode) link right at the top. 

> We usually recommend that articles be between 1000 to 1500 words but this one is ~3400. We would like to stick to this policy because there are only a small group of folks doing content reviews and it's hard for them to scale when content is really lengthy. What has tended to happen in the past is really long articles get stuck in the content review stages for a long time and we want to avoid that.

@robdodson If the concern here is review constraints, please note that this has already been [reviewed](https://docs.google.com/document/d/1SfFa2PqVS4PzvoYgD7yUntiFBkyE_jdxdVJ1ehkiKck/edit?usp=sharing) by lzbylut@ and @rowan-m.

Reply:
I guess there are two concerns:

1. Content that's >1000 words is harder for users to read quickly. Often times folks will open an article and quickly scroll to the bottom to see if they have time to read it all. If it looks like a lot the worry is someone might skip it.
2. It adds to the workload for reviewers. I think if other folks have reviewed the content then that's definitely helpful, but that won't always be the case so we want to encourage folks to stick to those lengths when they can.

In the future I might suggest splitting the post into two parts if the history is important but takes a lot of time to explain. Having a multi-part post series on a topic seems cool to me.

In this instance I think we can move ahead with this PR because it's already really close to the finish line.

Reply:
> In the future I might suggest splitting the post into two parts if the history is important but takes a lot of time to explain. Having a multi-part post series on a topic seems cool to me.

Duly noted for the next time. On the plus side, this will be a one-stop shop for all things dark mode. 

> In this instance I think we can move ahead with this PR because it's already really close to the finish line.

Great! If any of the owners LGTMs, I can merge tomorrow my local time and we can go live later that say.